core/iwasm/include/wasm_export.h

```diff
@@ -212,6 +212,14 @@ typedef struct wasm_val_t {
 } wasm_val_t;
 #endif
 
+typedef enum {
+    WASM_LOG_LEVEL_FATAL = 0,
+    WASM_LOG_LEVEL_ERROR = 1,
+    WASM_LOG_LEVEL_WARNING = 2,
+    WASM_LOG_LEVEL_DEBUG = 3,
+    WASM_LOG_LEVEL_VERBOSE = 4
```

Comment:
Should also change all "BH_LOG_" related strings in the source code to "WASM_LOG_*".

Reply:
Yes, I was about to comment, do we want to do that or keep the internal name as `BH_LOG_LEVEL_*` and use `WASM_LOG_LEVEL_*` only when exposing?

Reply:
By re-introducing
```c
typedef enum {
    BH_LOG_LEVEL_FATAL = 0,
    BH_LOG_LEVEL_ERROR = 1,
    BH_LOG_LEVEL_WARNING = 2,
    BH_LOG_LEVEL_DEBUG = 3,
    BH_LOG_LEVEL_VERBOSE = 4
} log_level_t;
```
in `bh_log.h`.

Probably better to only keep `WASM_LOG_LEVEL_ *` and change all occurrences, I'll do that.

Reply:
Seems better to keep "BH_LOG_*" in bh_log.h and convert "WASM_LOG_*" to "BH_LOG_*" in wasm_runtime_set_log_level?